domains/Ibrahim.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "ibrahim's personal developer website",
+  "repo": "https://github.com/ibrahimdevx",
+  "owner": {
+    "username": "ibrahimdev",
+    "email": "ibrahimimran@northsixty.com"
+  },
+  "record": {
+    "A": "51.81.81.65"
+    "TXT": "static-site-verification=39ef1d198684ad37a3f962fb0a422aeb"
+  }
```

Comment:
You cannot have a `TXT` record, and the `A` record must be an array.

Reply:
Got it, I've fixed that using GitHub pages. My last host needed those extras